### DISCUSSION TITLE
No automatic mail when order is cancelled?

### DISCUSSION THREAD

**Discussion by bmaster001** [pretix/pretix#3866] [Q&A]
We're still testing and playing around, so excuse me if this is a dumb question :-) 
First I placed an order to test the payment system. Works perfect. Then I want to try how refunds work, so I cancelled that order (from the control panel as organizer, not as client). The refund is sent to Stripe, and the order is now shown as cancelled, but no mail seems to be sent out to the client. Is this normal?

**Comment by raphaelm** [comment]
Can you show the log of that order? Usually an email would be sent if not selected otherwise

**Reply by bmaster001** [reply]
Not sure where I can find such a log file. The only log file that I can find in the docker data folder for this container is pretix.log and this seems to only contain the api calls to Stripe... Maybe I have to look somewhere different?

**Reply by raphaelm** [reply]
Sorry, I was refering to the "order history" in the web view

**Reply by bmaster001** [reply]
Is this what you mean? I'm sorry, it's in Dutch and I blurred my email address. 
I checked the emailhistory as well, but that contains only the mails from the order and payment...
![image](https://github.com/pretix/pretix/assets/6114149/7ace5d8f-5a2c-477c-8639-9ba5028c78ca)

**Reply by bmaster001** [reply]
I figured it out!
1) In the order details, under the 'payments' box, I pressed 'create a refund'. There you get to choose the refund amount, and what shoud happen to the order. I choose to cancel the order, but then no mail is sent.
2) In the order details you have a button 'cancel order' on top. There you can choose to notify the customer by email, and in the next step you get to the refund screen. 

So both actions do basically the same, but with option 1 there is no option to notify the customer. I don't know if it's meant to be this way, but maybe it's a good idea to mention that somewhere in the process?
